_topic_map.yml

```diff
@@ -301,30 +303,47 @@ Topics:
 - Name: Installation configuration
   Dir: install_config
   Topics:
-  - Name: Installation methods for different platforms
-    File: installation-types
   - Name: Customizing nodes
     File: installing-customizing
   - Name: Creating a mirror registry for a restricted network
     File: installing-restricted-networks-preparations
-  - Name: Available cluster customizations
-    File: customizations
-    Distros: openshift-enterprise,openshift-webscale,openshift-origin
   - Name: Configuring your firewall
     File: configuring-firewall
-  - Name: Configuring a private cluster
-    Distros: openshift-enterprise,openshift-webscale,openshift-origin
-    File: configuring-private-cluster
 - Name: Troubleshooting installation issues
   File: installing-troubleshooting
 - Name: Support for FIPS cryptography
   File: installing-fips
   Distros: openshift-enterprise,openshift-webscale,openshift-dedicated,openshift-online
 ---
+Name: Post-installation configuration
+Dir: post_installation_configuration
+Distros: openshift-origin,openshift-enterprise,openshift-webscale
+Topics:
+- Name: Available cluster customizations
```

Comment:
I think this assembly should be incorporated into the `cluster-tasks` assembly. If you think it makes sense as a separate assembly, I think it needs a new title because "Cluster tasks" and "Available cluster customizations" seem too similar for the same book.

If you merge the assemblies, I suggest nesting the resource sections as H3s under a new H2 and keeping `Updating the global cluster pull secret` as an H2.

Reply:
Thank you for this suggestion. I have subsequently moved this content into the `cluster-tasks` assembly with the suggested heading structure. I have also updated the cross-reference to that in `installing/installing-preparing.adoc`.